run.sh

```diff
@@ -4,4 +4,4 @@ cd codebrag-ui
 nohup npm install && ./node_modules/.bin/grunt server &
 
 cd ../
-java -Dfile.encoding=UTF8 -Xmx3000M -Xss1M -XX:+CMSClassUnloadingEnabled -XX:MaxPermSize=1024m -jar sbt-launch.jar "~ container:start"
+java -Dfile.encoding=UTF8 -Xmx1590M -Xss1M -XX:+CMSClassUnloadingEnabled -XX:MaxPermSize=1024m -jar sbt-launch.jar "~ container:start"
```

Comment:
I was developing on a 32-bit machine, this has a memory limitation. But this should indeed not be part of the merge. I will revert this.